src/OutputWriters/output_construction.jl

```diff
@@ -43,10 +61,12 @@ end
 
 function construct_output(user_output::Union{AbstractField, Reduction}, grid, user_indices, with_halos)
     indices = output_indices(user_output, grid, user_indices, with_halos)
+    user_output = maybe_sliced_field(user_output, indices)
+
     return construct_output(user_output, indices)
 end
 
-construct_output(user_output::Field, indices) = view(user_output, indices...)
+construct_output(user_output::Field, indices) = user_output
```

Comment:
why is this fallback introduced?

Reply:
I am a bit confused about this `construct_output`. Why does a field output to a `view` while an `AbstractOperation` outputs to a field instead of a `view`? cc @glwagner